codegenerator/cli/templates/static/codegen/src/EventFetching.res

```diff
@@ -31,6 +33,51 @@ let rec getKnownBlockWithBackoff = async (~provider, ~blockNumber, ~backoffMsOnF
   | result => result
   }
 
+let getSuggestedBlockIntervalFromExn = {
+  let suggestedRangeRegExp = %re(`/retry with the range (\d+)-(\d+)/`)
```

Comment:
Is this our own error? Surely we can use a rescript exception or some sort of return type rather than parsing our own error?

Reply:
No, this is from the rpc provider